rnd/autogpt_builder/src/lib/autogpt-server-api/baseClient.ts

```diff
@@ -44,8 +44,8 @@ export default class BaseAutoGPTServerAPI {
     return await this._get("/blocks");
   }
 
-  async listGraphs(): Promise<GraphMeta[]> {
-    return this._get("/graphs");
+  async listGraphs(with_runs: boolean = false): Promise<GraphMeta[]> {
+    return this._get(`/graphs?with_runs=${with_runs}`);
```

Comment:
If it returns a different type depending on the parameter, pls differentiate the output type and add an overload.

Reply:
It does not return different type depending on the output.